x-pack/filebeat/Jenkinsfile.yml

```diff
@@ -64,38 +62,37 @@ stages:
                 - "macOS"
             parameters:
                 - "macosTest"
-            tags: true         ## for all the tags
         stage: extended
-    windows:
+    windows-2022:
```

Comment:
I am a bit curious about the change here from `windows` -> `windows-2022`. Since all the other top-level items are specific to a release (i.e. `windows-2019` like on L71), I am curious why there was a generic entry for `windows` to begin with. Was it some kind of default or catch-all, or was it just simply named this way for historical reasons? (Hopefully this question makes sense.)

Reply:
That's perfectly clear. Thank you for giving me the history!